p2p/kademlia/dht_test.go

```diff
@@ -366,9 +376,10 @@ func TestStoreAndFindLargeValue(t *testing.T) {
 
 	id1, _ := newID()
 	dht1, _ := NewDHT(ctx, getInMemoryStore(), &Options{
-		ID:   id1,
-		IP:   "127.0.0.1",
-		Port: 3000,
+		ID:             id1,
```

Comment:
i think u can try "github.com/stretchr/testify/suite" for the unit test, because i see many repeated hard code for many tests